spec/controllers/posts_controller_spec.rb

```diff
@@ -1139,21 +1139,18 @@
       expect(response).to render_template(:show)
     end
 
-    it "calculates audits" do
-      Reply.auditing_enabled = true
-      Post.auditing_enabled = true
-
-      replies = Audited.audit_class.as_user(post.user) do
+    it "calculates audits", versioning: true do
+      replies = Version.as_user(post.user) do
         create_list(:reply, 6, post: post, user: post.user)
       end
 
-      Audited.audit_class.as_user(post.user) do
+      Version.as_user(post.user) do
         replies[1].touch # rubocop:disable Rails/SkipsModelValidations
```

Comment:
Note to future self: this is the reason for the test failure, right here, Papertrail seems to be creating an audit for the touch. possibly an after_touch callback?